### PR TITLE
WinUI-3213 : Updated Culture in UG document for SfMaskedTextBox control

### PR DESCRIPTION
**Pull request by Vignesh-SF3580**
### Description
Updated culture support content for SfMaskedTextBox control.

### Solution Description
Updated the contents for culture support in masked text box control.
Added the gif images for the output.

### CODE REVIEW

File: winui/Masked-TextBox/Culture.md [hunk 1]
```diff
@@ -1,10 +1,76 @@
 ---
 layout: post
-title: Localization in WinUI MaskedTextBox control | Syncfusion
-description: Learn all about the localization in the MaskedTextBox (SfMaskedTextBox) control for the specific set of symbols such as '.', ',', '/', ':' and '$'.
+title: Culture in WinUI MaskedTextBox control | Syncfusion
+description: Learn all about the culture in the MaskedTextBox (SfMaskedTextBox) control for the specific set of symbols such as '.', ',', '/', ':' and '$'.
 platform: WinUI
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/Culture.md:5]: @Vignesh-SF3580 Learn all about how to set the different culture support for currency symbols, date separators, and decimal separators in the MaskedTextBox (SfMaskedTextBox) control.

File: winui/Masked-TextBox/Culture.md [hunk 2]
```diff
@@ -1,10 +1,76 @@
...
 control: SfMaskedTextBox
 documentation: ug
 ---
 
-# Localization in WinUI SfMaskedTextBox
\ No newline at end of file
+# Culture in WinUI MaskedTextBox
+
+Special symbols like currency symbols, date separators, decimal separators, and others are applied based on the specified culture.
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/Culture.md:12]: @Vignesh-SF3580 The MaskedTextBox allows you to set any culture support for currency symbols, date separators, time separators, group separators,  and decimal separators by using the below special symbols.

File: winui/Masked-TextBox/Culture.md [hunk 3]
```diff
@@ -1,10 +1,76 @@
...
+SfMaskedTextBox maskedTextBox = new SfMaskedTextBox();
+maskedTextBox.Width = "195";
+maskedTextBox.MaskType = MaskedTextBoxMaskType.Simple;
+maskedTextBox.Mask = "$ 0,000.00";
+maskedTextBox.Culture = new CultureInfo("fr-FR");
+
+{% endhighlight %}
+{% endtabs %}
+
+Now the ‘$’ will be localized to ‘€’; ‘.’ will be localized to ‘,’ and ‘,’ will be localized to ‘ ‘(single white space).
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/Culture.md:74]: @Vignesh-SF3580 Based on the France Culture,

File: winui/Masked-TextBox/Culture.md (hunk 1 above)
Comment (Vignesh-SF3580) [reply winui/Masked-TextBox/Culture.md:5]: Changed the description like this.

File: winui/Masked-TextBox/Culture.md (hunk 2 above)
Comment (Vignesh-SF3580) [reply winui/Masked-TextBox/Culture.md]: Changed the content.

File: winui/Masked-TextBox/Culture.md (hunk 3 above)
Comment (Vignesh-SF3580) [reply winui/Masked-TextBox/Culture.md]: Added the based on France culture in the content.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

**Pull request merged**